python/helpers/jailbreak_utils.py

```
@@ -0,0 +1,633 @@
+import asyncio
```

Ruff (0.11.9)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)